File: fastframe/src/main/java/com/code/fastframe/ciface/IBaseRecyclerViewCb.java
package com.code.fastframe.ciface;

import android.view.View;

/**
 * Created by dengshaomin on 2017/12/5.
 */

public interface IBaseRecyclerViewCb {

    void onHeadFootClickLister(View view, Object data, int position);

    void onRefresh(int currentState);
}

File: fastframe/src/main/java/com/code/fastframe/ciface/IBaseRecyclerViewCb.kt
package com.code.fastframe.ciface

import android.view.View

/**
 * Created by dengshaomin on 2017/12/5.
 */
interface IBaseRecyclerViewCb {
  fun onHeadFootClickLister(
    view: View?,
    data: Any?,
    position: Int
  )

  fun onRefresh(currentState: Int)
}